website/sidebars.js

```diff
@@ -153,6 +153,15 @@ module.exports = {
                     label: 'iOS',
                     id: 'feature-flag-tutorials/ios/implementing-feature-flags',
                 },
+                {
+                    type: 'category',
+                    label: 'Rust',
+                    link: {
+                        type: 'doc',
+                        id: 'feature-flag-tutorials/rust/implementing-feature-flags-rust',
+                    },
+                    items: [],
+                },
```

Comment:
Why is this a category? Shouldn't this just be a doc, like the iOS one above it?

```suggestion
                {
                    type: 'doc',
                    label: 'Rust',
                    id: 'feature-flag-tutorials/rust/implementing-feature-flags-rust',
                },
```

Reply:
This is because the examples #7572 were coming shortly after so I thought I'd make our lives easier

Reply:
Ah, right! Yeah, that makes sense. Ignore this, then 😄